thorlcr/graph/thgraphslave.hpp

```diff
@@ -70,15 +70,15 @@ class CEdgeProgress
             if (!hasStopped())
                 throw MakeActivityException(&owner, 0, "Starting without being stopped 1st");
             else
-                throw MakeActivityException(&owner, 0, "Started and stopped states both set");
+                count &= THORDATALINK_COUNT_MASK;
```

Comment:
this line is not necessary, as will fall through to line 76 that masks and sets.